stat11_backend/stat11/utils/match_team_data.py

```diff
@@ -4,22 +4,28 @@
 def get_match_team_data(match_id):
     match_team_data = []
     match_teams = Team.objects.filter(match__id=match_id)
-    
-    # print(match_teams)
-    # for i in match_teams:
-    #     print(vars(i))
 
     for index, team in enumerate(match_teams):
         team_runs = 0
+        team_wickets = 0
         team_extras = 0
+        team_balls = 0
         team_batter_scoreboard = BatterScoreboard.objects.filter(team__id=team.id)
         for scoreboard in team_batter_scoreboard:
+            team_balls += scoreboard.balls
             team_runs += scoreboard.runs
+            if scoreboard.exit_time is not None:
+                team_wickets += 1
         serializer = TeamNestedSerializer(team)
         data = serializer.data
         data['team_runs'] = team_runs
         team_extras=team.noball+team.wide+team.bye+team.legbye
         data['team_extras'] = team_extras
```

Comment:
You don't need to send team_extras seperately.....these fields will already be present in the serialized data of team

Reply:
It has been used in many places already. one extra attribute won't harm, let it be

Reply:
Marking it resolved as of now, will make changes if required